rfcs/packaging/package-structure.md

```diff
@@ -0,0 +1,145 @@
+
+Full details of how to handle versioning within the repo (especially where major release servicing branches are concerned) will be discussed in a separate RFC.
+
+### How to retire legacy components
+
+See open issues section.
+
+### Centralize communication channels with early adoption partners
+
+We should make a private Teams channel within Fluent Community for internal partners who are working closely with us to try out the preview components. (It's possible that this could be expanded to external partners in the future.) This would be a very direct way for partners to share feedback and get support (from us or each other) with preview components. It would also make it easier for the whole team to see the feedback rather than it being highly compartmentalized.
+
+## Discarded Solutions
+
+<!-- As you enumerate possible solutions, try to keep track of the discarded ones. This should include why we discarded the solution. -->
+
+### "Insiders" channel and package
+
+During an earlier meeting about this topic, we came up with the idea of an "insiders" release channel: a way that early adoption partners could consume components in preview, and related communication channels.
+
+We decided not to use the "insiders" naming since it has some specific connotations/expectations attached, and while what we have in mind is similar, we don't want people to feel like we've over-promised/under-delivered based on their previous experience with more formalized insider programs. However, the `@fluentui/react-preview` proposal and early adopter Teams channel proposal stemmed from this idea.
+
+## Open Issues
+
+<!-- Optional section, but useful for first drafts. Use this section to track open issues on unanswered questions regarding the design or proposal.  -->
+
+### Suite package(s) for only converged components
+
+While consumers are welcome to use individual component packages directly, we'll also add one or possibly two new suite packages which export _only_ converged components.
+
+#### Possibility: `@fluentui/react-preview`
+
+(this was the initial proposal)
+
+Sometime in the next couple months after version 8 release, add a `@fluentui/react-preview` package which exports _only_ converged components which have reached "80% ready"/preview status. Notes:
+
+- Exports will go away once they're moved to the suite
+  - "suite" here means `@fluentui/react`; promotion details discussed later
+- Document clearly from the start that the stability is lower, and people can consume it if they're willing to accept that minor (or prerelease?) versions may sometimes contain breaking changes (to be clearly noted in the changelogs)
+- Either 1st or 3rd parties can use this if they're willing to accept the extra risk/work
+- Readme should be kept updated with status of each component
+- If people prefer, they can use the individual preview component packages, but `@fluentui/react-preview` is the "one-stop shop" for previewing converged components
+
```

Comment:
We probably also need to list that, once something is deemed stable, it will "graduate" from the `@fluentui/react-preview` package, essentially not exporting it from there anymore.

Reply:
Yup. That was sort of mentioned in the first bullet in this section, but I updated it to be even more explicit.